### ISSUE TITLE
Improving training consistency

### ISSUE THREAD

**Issue by jyaacoub** [jyaacoub/MutDTA#112]
In order to make sure each model options are given a fair shot we need to make sure the training regime is the same for all of them when we go to make our comparisons. This means we need the **same exact splits for our cross validation.**

However for some models, the nature of their parameters means we have fewer data to work with (e.g.: aflow model is limited to proteins under 1200 in length due to computational reasons). So we have to be careful when making comparisons and account for this by only comparing against **models trained on the same subset of proteins as the aflow model.**

So basically we need two things:
- [x] #113
- [x] #115
